sycl/include/CL/sycl/backend_types.hpp

```diff
@@ -18,7 +18,7 @@
 __SYCL_INLINE_NAMESPACE(cl) {
 namespace sycl {
 
-enum class backend : char { host, opencl, level_zero, cuda };
+enum class backend : char { host, opencl, level_zero, cuda, all };
```

Comment:
This is bikeshedding, but "any" probably makes more sense than "all" in this context.

Reply:
'any' makes sense to me, too.
However, we already have device_type::all instead of device_type::any.
So, for consistency, I still to "all".